content/sales.rst

```diff
@@ -0,0 +1,196 @@
+  toll free / long distance numbers to provide)
+* Number of attendees (max 4 attendees incl 1 organiser and 1 co-organiser in
+  case or max 5 attendees if 1 organiser)
+* Platform
+* Protocols
+* List or criteria of requirements to know what to set up in advance
+
+
+Marketing / Promotional Materials
+=================================
+
+Promotions and marketing materials are mainly located in the main website.
+
+It should be as generic and non-technical as possible with links to the
+Documentation for more in-depth / technical information.
+
+To save materials, or to obtain past materials, the `chevah/design` repository
+is available.
+Please ask the team for access to the repository.
+
+
+Announcing a new release to the email list
+------------------------------------------
+
+After the website is updated and News item published, we send a newsletter:
+
+1. Go to Campaigns in Mailchimp.
+
+2. Select 'Replicate' besides 'NEW: ACME Release Announcement'.
+   If it is a security bugfix, use the Security Advisories email list.
+
+3. Select the News Announcements email list.
+
+4. Update the subject and email with the News text used to announce the
+   new release. You can use the text in the News article.
+
+5. Select Send. Before sending the final email, preview first by going
+   to 'Preview and Send' on the top menu. Select 'Send a test email'.
+
+
+Mock ups and designs for the website
+------------------------------------
+
+If a change involves a design or content addition (such as an image carousel
+in JS), it is a good idea to write/mock up the content first before coding.
+
+In this way, you can check to see what type of code work should be done to best
+communicate the content.
+
+Please go to the 'design' repository for sample images and screenshots to use
+and add your own samples.
+
+If raw HTML needs to be used, see if custom directives can be used such as:
+
+.. sourcecode:: bash
+
+    :call_for_action: Ready to install our product?
+    :call_for_action_link: /pricing/?utm_source=client&utm_campaign=clientbtn&utm_medium=btn#id1
+    :call_for_action_button: Ask for a trial
+
+For documentation pages, please do not add raw HTML as the format is designed
+to be converted into multiple other formats.
+
+
+Campaign/Design Outsourcing
+---------------------------
+
+Design and campaign work can be outsourced.
+Please contact the team for more details, including contact details of past
+freelancers.
+
+When planning for a campaign:
+
+* First collate the content for the campaign and send to the team
+
+* Once ready, create the initial brief
+
+* The finished materials provided should then be saved in the `chevah/design`
+  repository.
+
+
+Google Analytics
+----------------
+
+Certain parts of the website have Google Analytics tracking in order to track
+the results of a website campaign.
+
+The following are current active campaigns that include Google Analytics
```

Comment:
the styleguide should have generic info. this list is a good candidate for a wiki page... or maybe when you connect to analycits you will already see them